sonic-thermalctld/scripts/thermalctld

```diff
@@ -515,9 +524,13 @@ class TemperatureUpdater(logger.Logger):
         low_threshold = NOT_AVAILABLE
         high_critical_threshold = NOT_AVAILABLE
         low_critical_threshold = NOT_AVAILABLE
```

Comment:
certain vendors/platforms have three categories to measure temperature severity - minor, major and critical - and raise alarms and perform actions accordingly.
For such cases, NOS to define all three. 
Can we replace low_* with minor_* and add high_major_threshold and low_major_threshold?

Reply:
Discussed this in 10/21 Chassis Workgroup meeting. Can you please do the needful?

1st para in above comment: thermalctlD HLD needs to add section for Voltage, current environmental as as TODO section. Required for managing these sensors and their use-cases - impact on board, LC & FC power consumption calculation.

2nd para in above comment: PMON's thermalctlD section and code/PR to be updated to reflect this request/ask.

Reply:
We have a 'Future Items' task list in the HLD. Will add it as a to-do there. These would be enhancement items and as a community we can decide priority and ownership.